app/bibleview-js/src/components/TextEditor.vue

```diff
@@ -103,18 +113,50 @@ export default {
       title: 'Insert bible reference',
       result: async () => {
         const originalRange = document.getSelection().getRangeAt(0);
+        //Always add the "en" language, so at least the english parser always exists.
+        var en_parser = new bcv_parser;
+        var parsers = [en_parser]
+
+        if (features.features.refParser) {
+            //Get the active languages and create a bible reference parser for each language
+            const languages = get_languages()
+            languages.forEach(async (lang) => {
```

Comment:
I guess we want to wait for this to finish?

```suggestion
            await Promise.all(languages.map(async (lang) => {
```